index.html

```diff
@@ -1170,6 +1175,25 @@ <h2 id="functions">Function (uh, ahem) Functions</h2>
 });
 </pre>
 
+<p id="restParam">
+        <b class="header">memoize</b><code>_.memoize(function, [index=function.length-1])</code>
+        <span class="alias">Alias: <b>restArgs</b></span>
+        <br />
+        Similar to ES6's <a href="https://developer.mozilla.org/docs/Web/JavaScript/Reference/Functions/rest_parameters">rest param</a>,
+        <code>restParam</code> collects arguments passed to a function from
+        a given <code>index</code> as an array.
+      </p>
+      <pre>
+function multiply(multiplier, ...theArgs) {
+  return theArgs.map(function (element) {
+    return multiplier * element;
+  });
+}
```

Comment:
It'd probably be more helpful to show how a `restParam` function translates into ES6.